pkg/cli/rook_flexvolume_to_csi.go

```diff
@@ -0,0 +1,43 @@
+package cli
+
+import (
+	"log"
+	"os"
+
+	"github.com/replicatedhq/kurl/pkg/rook"
+	"github.com/spf13/cobra"
+	"k8s.io/client-go/kubernetes"
+	"sigs.k8s.io/controller-runtime/pkg/client/config"
+)
+
+func NewRookFlexvolumeToCSI(_ CLI) *cobra.Command {
+	var opts rook.FlexvolumeToCSIOpts
+
+	cmd := &cobra.Command{
+		Use:   "flexvolume-to-csi",
+		Short: "Converts Rook Flex volumes to Ceph-CSI volumes.",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			clientConfig := config.GetConfigOrDie()
+			clientset := kubernetes.NewForConfigOrDie(clientConfig)
+
+			logger := log.New(os.Stderr, "", 0)
```

Comment:
Does kurl normally log to stderr? I thought we used stdout?

Reply:
its a good question. i typically prefer a cli to log to stderr unless the output is a response to what is requested by the command. also the go log.Default() logger goes to stderr.

Reply:
since the cmd.Run function does not print anything I think it makes sense to log to stdout in this case, the error returned from run will go to stderr